src/wallet/message_store.cpp

```diff
@@ -1340,7 +1340,9 @@ bool message_store::check_for_messages(const multisig_wallet_state &state, std::
     }
   }
   std::vector<transport_message> transport_messages;
-  bool r = m_transporter.receive_messages(destinations, transport_messages);
+  if (!m_transporter.receive_messages(destinations, transport_messages)) {
+    return false;
+  }
```

Comment:
Styling remark. Insert a line break before the opening curly bracket for consistency.